lib/factory.rb

```diff
@@ -11,7 +11,7 @@ def self.new(*arguments, &block)
         raise ArgumentError, 'Excess arguments' if params.size > arguments.size
 
         arguments.each_with_index do |_value, index|
-          instance_variable_set("@#{arguments[index]}", params[index])
+          instance_variable_set("@#{_value}", params[index])
```

Comment:
Rubocop add `_` because the variable wasn't used, now it's used, so remove the `_`.